tests/platform_tests/api/watchdog.yml

```diff
@@ -62,3 +62,10 @@ x86_64-arista.*:
     valid_timeout: 10
     greater_timeout: 100
     too_big_timeout: 660
+
+# Celestica Seastone watchdog
+x86_64-cel_seastone-r0:
+  default:
+    valid_timeout: 10
```

Comment:
The `valid_timeout` is same as the default value. It is better to remove it here. Then the default value will be taken.

Reply:
updated https://github.com/Azure/sonic-mgmt/pull/3172/commits/b8aa1db282aac3d7082889f8c8eaaafc3686881b